batch/batch/file_store.py

```diff
@@ -39,6 +39,11 @@ def resource_usage_path(self, format_version, batch_id, job_id, attempt_id, task
             return f'{self.batch_log_dir(batch_id)}/{job_id}/{task}/resource_usage'
         return f'{self.batch_log_dir(batch_id)}/{job_id}/{attempt_id}/{task}/resource_usage'
 
+    def jvm_profile_path(self, format_version, batch_id, job_id, attempt_id, task):
+        if not format_version.has_attempt_in_log_path():
+            return f'{self.batch_log_dir(batch_id)}/{job_id}/{task}/jvm_profile'
```

Comment:
Suggestion: Could we end this in`.html` so that if someone wants to download from GCS it's more easily auto-opened in a browser?